pull-requests.md

```diff
@@ -139,6 +139,7 @@ Functional tests typically follow this pattern:
 * **Then** the end result should be X (and Y and Z).
 
 Before running the functional tests, you'll need a MySQL (or MariaDB) user called `wp_cli_test` with the password `password1` that has full privileges on the MySQL database `wp_cli_test`.
+To override these credentials checkout [database credentials of wp-cli-tests](https://github.com/wp-cli/wp-cli-tests#the-database-credentials)
```

Comment:
Only realized after approving, but `checkout` is misleading here, as that would imply a `git checkout` command.
```suggestion
To override these credentials you can make use of the [database credentials constants of wp-cli-tests](https://github.com/wp-cli/wp-cli-tests#the-database-credentials)
```